lib/IDE/CodeCompletionResultType.cpp

```diff
@@ -234,21 +245,33 @@ const USRBasedType *USRBasedType::fromType(Type Ty, USRBasedTypeArena &Arena) {
       auto Conformances = Nominal->getAllConformances();
       Supertypes.reserve(Conformances.size());
       for (auto Conformance : Conformances) {
+        if (isa<InheritedProtocolConformance>(Conformance)) {
+          // Skip inherited conformances; we'll collect them when we visit the
+          // superclass.
+          continue;
+        }
         if (Conformance->getDeclContext()->getParentModule() !=
             Nominal->getModuleContext()) {
           // Only include conformances that are declared within the module of the
           // type to avoid caching retroactive conformances which might not
           // exist when using the code completion cache from a different module.
           continue;
         }
-        if (Conformance->getProtocol()->isSpecificProtocol(KnownProtocolKind::Sendable)) {
+        if (Conformance->getProtocol()->isSpecificProtocol(KnownProtocolKind::Sendable) ||
+            Conformance->getProtocol()->getInvertibleProtocolKind()) {
           // FIXME: Sendable conformances are lazily synthesized as they are
           // needed by the compiler. Depending on whether we checked whether a
           // type conforms to Sendable before constructing the USRBasedType, we
           // get different results for its conformance. For now, always drop the
           // Sendable conformance.
+          //
+          // FIXME: Copyable and Escapable conformances are skipped because the
+          // USR mangling produces the type 'Any' for the protocol type.
           continue;
         }
+        LLVM_DEBUG(llvm::dbgs() << "Adding conformed protocol "
+                                << Conformance->getProtocol()->getName()
+                                << "\n";);
```

Comment:
Are these debug statements intentionally left?

Reply:
They're only active when you run with `-Xllvm -debug-only=CodeCompletionResultType`